src/gstools_cython/__init__.py

```diff
@@ -0,0 +1,33 @@
+"""
+Purpose
+=======
+
+GeoStatTools is a library providing geostatistical tools
+for random field generation, conditioned field generation,
+kriging and variogram estimation
+based on a list of provided or even user-defined covariance models.
+
+This package provides the Cython backend implementations for GSTools.
+
+Subpackages
+===========
+
+.. autosummary::
+   :toctree: api
+
+    field
+    krige
+    variogram
+"""
+
+# Hooray!
```

Comment:
Everytime I see this comment I ask myself what the story behind it is...

Reply:
It was commit number 1000 in the gstools repo ;-)